### DISCUSSION TITLE
github desktop vs browser

### DISCUSSION THREAD

**Discussion by siofi69** [community/community#136199] [New to GitHub] [answered]
### Select Topic Area

Question

### Body

I'm quite new to GitHub as it was I was recently requested to use it by my company. Reddit and other online sources aren't giving me clarity on whether the desktop version is better or just using it on browser. Could someone explain it briefly

**Comment by master-coco** [comment, marked as answer]
I personally just use Github browser more often as Github hasa integration with VS Code and copilot so you can just add, commit and push your code from there as well as clone repos and much more 
I was told the desktop application was slightly easier to use but try both for a bit and choose from your preference is what I would say

**Reply by siofi69** [reply]
Thank you for the quick response. I shall do so and let you know how it works
